eox_tenant/models.py

```diff
@@ -65,6 +66,99 @@ def get_microsite_for_domain(cls, domain):
         return microsites[0] if microsites else None
 
 
+class TenantConfigManager(models.Manager):
+    """
+    Custom managaer for Tenant Config model.
+    """
+    def get_configurations(self, domain):
+        """
+        Execute optimized query to get site configurations.
+        """
+        from django.db import connection
```

Comment:
Is this to avoid a circular import? Since in the line 8 is importing from django.db too. @diegomillan